### DISCUSSION TITLE
MacVim Release 173

### DISCUSSION THREAD

**Discussion by ychin** [macvim-dev/macvim#1252] [Releases]
Updated to Vim 9.0.0065.

MacVim is now updated to Vim 9!

I just want to apologize for the infrequent update cadence for MacVim over the last year and a half. I have not had much time to work on MacVim, which made catching up harder, leading to a bad feedback loop. I would like to move back to a more regular release cadence to keep MacVim more updated relative to Vim, and to be able to start working on macOS 12/13 integrations as well as fixing outstanding bugs and merging the various pull requests that have been waiting to be merged for a while.  Please don't hesitate to poke on GitHub if MacVim starts falling behind again.

General
====================

- MacVim now supports the `+sodium` feature, which allows you to use the newer "xchacha20" encryption method for encryption. See `:help cryptmethod`. #1225

Misc
====================

- Scripting languages versions:
    - Ruby is now built against 3.1, up from 3.0.

Compatibility
====================

Requires macOS 10.9 or above.

Script interfaces have compatibility with these versions:

- Lua 5.4
- Perl 5.18
- Python2 2.7
- Python3 3.9
- Ruby 3.1

<hr /><em>This discussion was created from the release <a href='https://github.com/macvim-dev/macvim/releases/tag/snapshot-173'>MacVim Release 173</a>.</em>

**Comment by MaurizioLoreti** [comment]
Do you plan to release separate versions for x86_64 and M1/M2 chipsets? Or are you relying on Rosetta?

**Reply by ychin** [reply]
MacVim is released as an universal app. You just need to download the dmg and it will use the right version.

**Reply by ychin** [reply]
This was done in r170 https://github.com/macvim-dev/macvim/releases/tag/snapshot-170

**Reply by MaurizioLoreti** [reply]
Thank you; sorry, I missed it.
